.github/workflows/deploy_terraform.yml

```diff
@@ -43,21 +51,8 @@ jobs:
     steps:
       - name: Check Out Changes
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
-      - name: Connect to VPN and login to Azure
-        uses: ./.github/actions/vpn-azure
-        with:
-          env-name: ${{ needs.pre_job.outputs.env_name }}
-          tls-key: ${{ secrets.TLS_KEY }}
-          ca-cert: ${{ secrets.CA_CRT}}
-          user-crt: ${{ secrets.USER_CRT }}
-          user-key: ${{ secrets.USER_KEY }}
-          sp-creds: ${{ env.AZURE_CREDENTIALS }}
-          tf-auth: true
       - name: Collect Terraform stats
```

Comment:
We need the `- name: Connect to VPN and login to Azure` step to authenticate tF to Azure